client/i18n/locales/espanol/links.json

```diff
@@ -22,7 +22,7 @@
     "contribute": "https://contribute.freecodecamp.org/#/i18n/espanol/index",
     "forum": "https://forum.freecodecamp.org/c/espanol/",
     "news": "https://freecodecamp.org/espanol/news/",
-    "podcast": "https://freecodecamp-en-espanol.transistor.fm/"
+    "podcast": "https://www.freecodecamp.org/espanol/news/el-podcast-de-freecodecamp-en-espanol/"
```

Comment:
just curious - would a spotify (or other podcast platform) link be preferable here? I'm fine with it if this is what we want.

Also, the podcasts look great @RafaelDavisH 🚀 Even though I can't understand them.

Reply:
Spotify it would be fine as well, but I do have an article with all the seasons and its episodes listed. I thought it might help drive some traffic to our ES publication.

Reply:
Sounds good 👍